client/settings/payment-method-icon/index.js

```diff
@@ -9,16 +9,9 @@ import classNames from 'classnames';
  * Internal dependencies
  */
 import './style.scss';
-import paymentMethodsMap from '../../payment-methods-map';
 
-const PaymentMethodIcon = ( { name, showName } ) => {
-	const paymentMethod = paymentMethodsMap[ name ];
-
-	if ( ! paymentMethod ) {
-		return <></>;
-	}
-
-	const { label, icon: Icon } = paymentMethod;
+const PaymentMethodIcon = ( { Icon, label } ) => {
```

Comment:
This component used to accept an id as a `name` prop, that represented a key in `paymentMethodsMap`. GooglePay/ApplePay, Stripe Link, WooPay are not defined in `paymentMethodsMap`.
I refactored this component so it accepts any `Icon` and any `label`, instead of accepting a payment method ID.
This way, we can use any type of icon, not just the ones defined in `paymentMethodsMap`.